src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
 
         // Validate input
         if (input == null || input.isEmpty()) {
```

Comment:
There is a typo in the print statement. It should be 'Concatenated string: ' not 'Cotenated string: '